### PR TITLE
platform: add Radxa ROCK 5A platform support

### PR DESCRIPTION
**Pull request by nascs**

### CODE REVIEW

**Comment by nascs** [comment]
Hi @tingleby, can you please take a look at the pr? I've finished testing it.

File: api/mraa/types.h [hunk 1]
```diff
@@ -70,6 +70,7 @@ typedef enum {
     MRAA_INTEL_ILK = 25,            /**< Intel Learning Kit */
     MRAA_SIEMENS_IOT2050 = 26,      /**< Siemens IOT2050 board */
     MRAA_RADXA_ROCK_3C = 27,        /**< Radxa ROCK 3 Model C */
+    MRAA_RADXA_ROCK_5A = 28,         /**< Radxa ROCK 5 Model A */
```
Comment (tingleby) [inline api/mraa/types.h]: we just merged another platform, could you update the enum with a new value?

**Review by tingleby — CHANGES_REQUESTED** [review]
Looks good, can you just update the enum with the next value, we merged a platform just now, sorry

File: api/mraa/types.h (hunk 1 above)
Comment (nascs) [reply api/mraa/types.h]: I just updated the values. Thank you so much for taking the time to review the pr.

**Comment by nascs** [comment]
Hi @tingleby,  I've changed the value. and I've resolved the conflict, can you look at it again?

**Pull request merged**